yuptoo/processor/report_processor.py

```diff
@@ -177,27 +178,14 @@ def extract_report_slices(report_tar, request_obj):
                                 continue
 
                             # Here performace can be improved by using Async thread
-                            report_files.append(report_slice_json)
-                return report_files
+                            process_report_slice(report_slice_json, request_obj)
+                            del report_slice_json
```

Comment:
What is the intention here? Do you want to delete json object? GC does automatic deallocation, right?

Reply:
As discussed internally, After first commit in this PR QE reported that Yuptoo is still consuming the same memory as earlier. So I tried to manually delete object and invoked the garbage collector, But then we found that PromQL query which we are looking is incorrect. 
For now I will revert this changes and will ask QE to test again.